code/modules/onestar/vending.dm

```diff
@@ -0,0 +1,46 @@
+/obj/machinery/vending/one_star
+	name = "One Star Vendor"
+	desc = "A vendor of the One Star variety."
+	icon = 'icons/obj/machines/one_star/vending.dmi'
+	icon_state = "vendor_guns"
+	icon_vend = "vendor_printing"
+	spawn_tags = SPAWN_TAG_OS_VENDOR
+	spawn_blacklisted = TRUE
+	bad_type = /obj/machinery/vending/one_star
+	//product_slogans = "Usually no carcinogens!;Best sports!;Become the strongest!"
+	//product_ads = "Strength!;Cheap!;There are contraindications, it is recommended to consult a medical specialist."
+	vendor_department = DEPARTMENT_OFFSHIP
+	alt_currency_path = /obj/item/stack/os_cash
+
+/obj/machinery/vending/one_star/Initialize()
+	. = ..()
+	set_light(1.4, 1, COLOR_LIGHTING_CYAN_BRIGHT)
+	earnings_account = department_accounts[DEPARTMENT_OFFSHIP]
+
+/obj/machinery/vending/one_star/guns
+	desc = "A vendor of the One Star variety. This one sells firearms of the One Star variety."
```

Comment:
There has to be a better way to word that to avoid repetition.
Something like "A One Star firearms vendor" "A One Star food vendor" and so on

Reply:
These are placeholders for Valo to rewrite.

Reply:
Ok